scripts/make_cross_border_flows.py

```diff
@@ -0,0 +1,110 @@
+# SPDX-FileCopyrightText: Contributors to PyPSA-Eur <https://github.com/pypsa/pypsa-eur>
+#
+# SPDX-License-Identifier: MIT
+"""
+Export energy flows (imports and exports) at each snapshot.
+"""
+
+import logging
+
+import numpy as np
+import pandas as pd
+import pypsa
+
+from scripts._helpers import configure_logging, set_scenario_config
+
+logger = logging.getLogger(__name__)
+
+
+def extract_flows_timeseries(n: pypsa.Network) -> pd.DataFrame:
+    """
+    Extract all lines and links flows at each snapshot.
```

Comment:
don't we only want to see cross border flows, i.e. if `bus0` / `bus1` are different regions? I think atm it will create a list of ALL busses, so also, for example, `H2` production through electrolysis within the same node